packages/devtools_app/.vscode/launch.json

```diff
@@ -60,5 +60,12 @@
             "type": "dart",
             "request": "attach",
         },
+        {
+            "name": "standalone_ui/property_editor_sidebar",
+            "request": "launch",
+            "type": "dart",
+            "program": "test/test_infra/scenes/standalone_ui/property_editor_sidebar.stager_app.g.dart",
```

Comment:
won't this launch target only work now if you've opened the devtools_app folder directly in VS code? We still want this launch target to work if someone has opened the main devtools/ folder.

If this fixes a bug, then maybe we actually have a bug with how package roots are detected for the inspector?

Reply:
the problem may be that the tool/flutter-sdk directory is getting included in the inspector package roots detection (if that is the Flutter SDK being used to build your app)

Reply:
Hmm that likely is the problem, I see my pub root directories are:

![Screenshot 2025-01-24 at 1 40 14 PM](https://github.com/user-attachments/assets/b0d9a780-e112-48d0-b843-4ef0cc24ee5b)

Because `tool/flutter-sdk` is under `dev/devtools`, everything there is included. Deleting `/Users/elliottbrooks/dev/devtools` so that the only entry is `/Users/elliottbrooks/dev/devtools/packages` fixes this.

I'll add details to the bug, and just have this PR add the launch config to devtools_app (it was annoying anyways that I could only launch the property editor from devtools root)

Reply:
Oh actually I think I must have set that at some point 🤦‍♀️ When I delete both pub roots, and then use the auto-refresh button, it correctly adds only `/Users/elliottbrooks/dev/devtools/packages`. So I think we're okay here